website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -2,28 +2,114 @@
+
+The body allows for two types of content: `text` and `generic`.
+
+![Showing two different kind of body content types, one as text and another as generic yielded content](/assets/components/application-state/application-state-body-content-types.png)
+
+
+## Footer
+The Application State supports up to three actions, including [Dropdown](/components/dropdown), [Standalone Link](/components/link/standalone) and [Button](/components/button). Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
+
+![A button set showing a dropdown, secondary button and stand alone link](/assets/components/application-state/application-state-footer-action-types.png)
+
+### Using buttons or links
+
+Buttons, along with links, are used to emphasize the importance of the action in the Application State with visual hierarchy, and are the most common actions used in the footer.
+
+!!! Dont
+
+When there is an empty state that occupies the majority of the page, do not display two exact actions at the same time in different areas of the UI. In this example, there is a primary button in the Page Header and in the Application State. 
+
+![Showing an empty state with a primary button and a page header with a primary button](/assets/components/application-state/application-state-empty-state-dont-duplicate-buttons.png)
+
+!!!
+
+!!! Do
+
+Instead, use the Application State as the only means of drawing attention to the primary action.
+
+![Showing an empty state with a primary button with a page header with out a primay button](/assets/components/application-state/application-state-empty-state-do-keep-one-primary-cta.png)
+
+!!!
+
+### Using dropdowns
+
+Dropdowns can be used as actions in the footer in rare cases. Limit dropdowns to one per Application State.
+
+![Showing an empty state with a primary button with a page header with out one](/assets/components/application-state/application-state-dropdown-actions.png)
+
+## Width constraints
+
+The Application State’s content has a max width of 480 pixels. This is done for better readability, ensuring that the max character count is close to 70 characters per line.
+
+## Examples
+
+Here are some common use cases for the Application State, however, it is not limited to just these two examples.
+
+### Error state
+
+Error states are used when the application encounters an issue or error during its operation. It shows the associated error code, icon, messages and actions to help users find a solution.
+
+![Showing an example of an error state with a 404 error code and two links](/assets/components/application-state/application-state-error-state.png)
+
+### Empty state
+
+Empty states are used when a user has yet to create an object. Illustrations are placed using the `media` slot to further elevate the experience and express the purpose of the object.
```

Comment:
I'm not actually sure on this, would benefit from a second opinion, but saying "are used" reads weird to me because they aren't necessarily an intentional use, more of a fallback for when there isn't anything.

```suggestion
An empty state occurs when a user has yet to create an object. Illustrations are placed using the `media` slot to further elevate the experience and express the purpose of the object.
```

Reply:
Agreed, updating.